app/src/main/java/org/mozilla/fenix/home/topsites/TopSiteItemViewHolder.kt

```diff
@@ -30,8 +36,10 @@ import org.mozilla.fenix.home.sessioncontrol.TopSiteInteractor
 import org.mozilla.fenix.settings.SupportUtils
 import org.mozilla.fenix.utils.view.ViewHolder
 
+@SuppressLint("ClickableViewAccessibility")
```

Comment:
Did we need this top level suppression since we suppressed it below?

Reply:
I was getting a lint error with the suppression being on the lambda :/

> /Users/nbond/Documents/firefox-android/fenix-fork/app/src/main/java/org/mozilla/fenix/home/topsites/TopSiteItemViewHolder.kt:73:35: Annotation with parameter(s) should be placed on a separate line prior to the annotated construct (cannot be auto-corrected) (annotation)